docs/resources/service_connector.md

```diff
@@ -29,6 +29,8 @@ resource "zenml_service_connector" "gcp_connector" {
       "auth_uri": "https://accounts.google.com/o/oauth2/auth",
       "token_uri": "https://oauth2.googleapis.com/token"
     })
+
+    verify = true
```

Comment:
Verify isn't part of the configuration; it's part of the resource itself, alongside the configuratio,n not inside it